docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -227,6 +258,23 @@ Running the ``devel`` branch from a clone
 
 .. _shell_completion:
 
+Confirming your installation
+============================
+
+You can test that Ansible is installed correctly by checking the version:
+
+.. code-block:: console
+
+    $ ansible --version
+
+The version displayed by this command is for the associated ``ansible-core`` package that has been installed.
+
+To check the version of the ``ansible`` package that has been installed:
+
+.. code-block:: console
+
+    $ ansible-community --version
```

Comment:
@sivel is `ansible-community` a thing? O_0

Reply:
apparently.  I saw some note of it, and then decided to look into it